cloudformation/media-atom-maker-dev.yml

```diff
@@ -0,0 +1,53 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "Media atom maker development"
+Parameters:
+  DynamoDBTablePrefix:
+    Description: "DynamoDB Table prefix"
+    Type: "String"
+    Default: "media-atom-maker"
+Resources:
+  MediaAtomGroup:
+    Type: "AWS::IAM::Group"
+    Properties:
+      Policies:
+        - PolicyName:   "Group-Policy"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: "Allow"
+                Action: ["dynamodb:*"]
+                Resource:
+                  - Fn::Join:
+                    - ""
+                    -
```

Comment:
can do the `Fn::Join` on one line to avoid this annoying feature of yaml:

```yaml
Resource:
  Fn::Join: ['', ['arn:aws:dynamodb:', {Ref: AWS::Region}, ':', {Ref: AWS::AccountId}, ':table/', {Ref: AWS:: MediaAtomsDynamoTable} ]]
```